helperAPI.py

```diff
@@ -505,6 +512,24 @@ def getDriver(DOCKER=False):
                 service=EdgeService(),
                 options=options,
             )
+            """
+             #! What I added
+            chrome_options = Options()
+            chrome_options.add_argument('--ignore-certificate-errors')
+            chrome_options.add_argument('--ignore-ssl-errors')
+            chrome_options.add_argument('--disable-blink-features=AutomationControlled')
+            chrome_options.add_argument('--disable-notifications')
+            chrome_options.add_argument('--disable-popup-blocking')
+            chrome_options.add_argument('--disable-infobars')
+            chrome_options.add_argument('--disable-extensions')
+            chrome_options.add_argument("--no-sandbox")
+            chrome_options.add_argument("--disable-dev-shm-usage")
+            chrome_options.add_argument("--incognito")
```

Comment:
is there a specific reason to use incognito

Reply:
I got tired of the save passwords popups at every login